.github/workflows/pypi-publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish package to PyPi
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  push:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
Only runs for vX.X.X tags (ignore harmony-chart-X.X.X)

Reply:
Might be easier to just put it in a separate repo in the future, but this seems fine for now. I just wonder if we should use tags like `harmony-plugin-X.X.X` instead of just `vX.X.X`? I think that would be more clear.

Reply:
Sure, now that versions will be available on PyPI, we can name it as we want. Let me know if I can trigger the workflow on a PR event temporarily to make sure everything is working and to have a first version on PyPI

Reply:
Yes, go ahead and try it please :)